_posts/2020-04-17-snowplow-ios-tracker-1.3.0-released/index.md

```diff
@@ -0,0 +1,309 @@
+
+The GDPR context is enabled by calling the `setGdprContextWithBasis:documentId:documentVersion:documentDescription:` method of the tracker builder or by calling the `enableGdprContextWithBasis:documentId:documentVersion:documentDescription:` method once the tracker has been initialised.
+
+It is called as follows:
+
+{% highlight objc %}
+SPTracker *tracker = [SPTracker build:^(id<SPTrackerBuilder> builder) {
+    [builder setEmitter:emitter];
+    ...
+    [builder setGdprContextWithBasis:SPGdprProcessingBasisConsent
+                          documentId:@"someId"
+                     documentVersion:@"0.1.0"
+                 documentDescription:@"a demo document description"];
+}];
+{% endhighlight %}
+
+<h2 id="spm">3. Swift Package Manager support</h2>
+
+The Swift Package Manager is the official Apple's dependency manager designed for Swift libraries. Since Xcode 11 SPM is compatible with the iOS build system contributing to enlarge the mobile developers interst around this new dependency manager that will compete against the largely longtime used Cocoapods and Carthage.
+Thanks to the important contribute of [@vauxall][contributor] we have introduced the support of Swift Package Manager for our iOS tracker ([#474][474]). At the moment it supports only the iOS platform due to some limitations that we will resolve in the future releases, but it still is a great adding for our mobile tracker and a great contribute from our active community.
+
+To install Snowplow Tracker with SPM:
+1. in your Xcode select File > Swift Packages > Add Package Dependency
+2. set the repository: https://github.com/snowplow/snowplow-objc-tracker
+3. add the modules to your project's targets on General tab > _Frameworks, Libraries & Embedded Content_ section
+ 
+<h2 id="deprecated">4. Deprecated methods</h2>
+
+On this release we marked few methods as deprecated:
+
+- SPEvent - `eventId`: This shouldn't be used as it can cause ambiguity about event duplication. That will be set only by the tracker. If your code set its own eventId we suggest to record your custom event ID in a custom context associated to all the events.
+
+- SPEvent - `timestamp`: In future major versions we will remove the ability to override the timestamp of the event. That will be set only by the tracker. A custom timestamp can be eventually added using a different method or parameter in a future release.
+
+- SPEvent - `addDefaultParamsToPayload:`: This shouldn't be used as it's an internal method used by the tracker and it can be subject of future changes. 
+
+Other few methods can be set as deprecated because mostly related to internal logic of the tracker. We strongly suggest to avoid use of methods marked deprecated as they will be substituted or removed in the next major version.
```

Comment:
```suggestion
Other methods have also been marked as deprecated because they are related to internal logic of the tracker. We strongly suggest avoiding the use of these methods marked deprecated as they will be substituted or removed in the next major version.
```